### PR TITLE
chore(readme.md): add deprecation notice

### LINKED ISSUE #18: Deprecate this module
### Description

New module has been created that replaces this [here](https://commercetools.github.io/nodejs/cli/inventories-exporter.html)

Deprecate here and on NPM

### PR DESCRIPTION
**Pull request by daern91**
resolves #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,14 @@
 ![commercetools logo](https://cdn.rawgit.com/commercetools/press-kit/master/PNG/72DPI/CT%20logo%20horizontal%20RGB%2072dpi.png)
-
+⛔ [Deprecated]
 # Stock export
```
Comment (wizzy25) [inline README.md]: Perhaps you can also indicate that the name is changed in the new repo to Inventory export:

# Stock export (Inventory export)

**Review by wizzy25 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,66 +1,14 @@
...
-Stocks exported in CSV are stored in a single file
-
-```csv
-sku,quantity,channel
-```
-
-## Contributing
-In lieu of a formal styleguide, take care to maintain the existing coding style. Add unit tests for any new or changed functionality. Lint and test your code using [Grunt](http://gruntjs.com/).
-More info [here](CONTRIBUTING.md)
+This repo has been deprecated and the code migrated to this [repo](https://github.com/commercetools/nodejs).
```
Comment (LEQADA) [inline README.md]: I would put it directly after `⛔ [Deprecated]` . Something like 

> ⛔ [Deprecated] instead use [nodejs](https://github.com/commercetools/nodejs) (see examples [some url]())

File: README.md [hunk 3]
```diff
@@ -1,67 +1,20 @@
...
-```
-
-## Usage
-
-General command line options can be seen by simply executing the command `node lib/run`.
-```
-node lib/run
-
-  Usage:     --projectKey <project-key> --clientId <client-id> --clientSecret <client-secret>
+This repo has been deprecated and the code migrated to this [repo](https://github.com/commercetools/nodejs).
```
Comment (LEQADA) [inline README.md:9]: Don't we have any examples of usage? Shouldn't we help a user to migrate when we deprecate something?

Comment (daern91) [reply README.md:9]: @LEQADA, I think the link to the documentation should be enough help, no?
`The documentation about this module can be found [here](https://commercetools.github.io/nodejs/cli/inventories-exporter.html)`

File: README.md [hunk 4]
```diff
@@ -1,67 +1,20 @@
 ![commercetools logo](https://cdn.rawgit.com/commercetools/press-kit/master/PNG/72DPI/CT%20logo%20horizontal%20RGB%2072dpi.png)
 
-# Stock export
+## ⛔ [Deprecated], please use the [nodejs repo](https://github.com/commercetools/nodejs) instead
 
-[![NPM](https://nodei.co/npm/sphere-stock-export.png?downloads=true)](https://www.npmjs.org/package/sphere-stock-export)
-
-[![Build Status](https://secure.travis-ci.org/sphereio/sphere-stock-export.png?branch=master)](http://travis-ci.org/sphereio/sphere-stock-export) [![NPM version](https://badge.fury.io/js/sphere-stock-export.png)](http://badge.fury.io/js/sphere-stock-export) [![Coverage Status](https://coveralls.io/repos/sphereio/sphere-stock-export/badge.svg?branch=master)](https://coveralls.io/r/sphereio/sphere-stock-export?branch=master) [![Dependency Status](https://david-dm.org/sphereio/sphere-stock-export.svg)](https://david-dm.org/sphereio/sphere-stock-export) [![devDependency Status](https://david-dm.org/sphereio/sphere-stock-export/dev-status.svg)](https://david-dm.org/sphereio/sphere-stock-export#info=devDependencies)
+# Stock export (Inventory export)
```
Comment (LEQADA) [inline README.md]: Stock export (Inventories Exporter)

**Review by wizzy25 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,67 +1,20 @@
...
-
-General command line options can be seen by simply executing the command `node lib/run`.
-```
-node lib/run
-
-  Usage:     --projectKey <project-key> --clientId <client-id> --clientSecret <client-secret>
+This repo has been deprecated and the code migrated to this [repo](https://github.com/commercetools/nodejs).
 
-  Options:
+**NOTE:** It has also been renamed from `Stock export` to `Inventory Export`
```
Comment (LEQADA) [inline README.md]: **NOTE:** It has also been renamed from `Stock export` to `Inventories Exporter `

File: README.md (hunk 3 above)
Comment (LEQADA) [reply README.md:9]: Thanks for the link. I thought those links are still old

File: README.md (hunk 5 above)
Comment (daern91) [reply README.md]: Good find, danke schön @LEQADA! Updated again now. Please have another look.

**Review by LEQADA — APPROVED** [review]
LGTM 🙂

**Pull request closed without merging**